docs/src/use/configure/language-options.md

```diff
@@ -5,129 +5,67 @@ eleventyNavigation:
-```
-
-This enables Node.js and Mocha environments.
+The JavaScript ecosystem has a variety of runtimes, versions, extensions, and frameworks. Each of these can have different supported syntax and global variables. ESLint lets you configure language options specific to the JavaScript used in your project, like custom global variables. You can also use plugins to extend ESLint to support your project's language options.
 
-### Using configuration files
+## Specifying JavaScript Options
 
-To specify environments in a configuration file, use the `env` key. Specify which environments you want to enable by setting each to `true`. For example, the following enables the browser and Node.js environments:
+ESLint allows you to specify the JavaScript language options you want to support. By default, ESLint expects the most recent stage 4 ECMAScript syntax and ECMAScript modules (ESM) mode. You can override these settings by using the `languageOptions` key and specifying one or more of these properties:
 
-```json
-{
-    "env": {
-        "browser": true,
-        "node": true
-    }
-}
-```
+* `ecmaVersion` (default: `"latest"`) - Indicates the ECMAScript version of the code being linted, determining both the syntax and the available global variables. Set to `3` or `5` for ECMAScript 3 and 5, respectively. Otherwise, you can use any year between `2015` to present. In most cases, we recommend using the default of `"latest"` to ensure you're always using the most recent ECMAScript version.
+* `sourceType` (default: `"module"`) - Indicates the mode of the JavaScript file being used. Possible values are:
+    * `module` - ESM module (invalid when `ecmaVersion` is `3` or `5`). Your code has a module scope and is run in strict mode.
+    * `commonjs` - CommonJS module (useful if your code uses `require()`). Your code has a top-level function scope and runs in non-strict mode.
+    * `script` - non-module. Your code has a shared global scope and runs in non-strict mode.
 
-Or in a `package.json` file
+Here's an example configuration file you might use when linting ECMAScript 5 code:
 
-```json
-{
-    "name": "mypackage",
-    "version": "0.0.1",
-    "eslintConfig": {
-        "env": {
-            "browser": true,
-            "node": true
+```js
+// eslint.config.js
+export default [
+    {
+        languageOptions: {
+            ecmaVersion: 5,
+            sourceType: "script"
         }
     }
-}
-```
-
-And in YAML:
-
-```yaml
----
-  env:
-    browser: true
-    node: true
+];
 ```
 
-### Using a plugin
+## Specifying Parser Options
 
-If you want to use an environment from a plugin, be sure to specify the plugin name in the `plugins` array and then use the unprefixed plugin name, followed by a slash, followed by the environment name. For example:
+If you are using the built-in ESLint parser, you can additionally change how ESLint interprets your code by specifying the `languageOptions.parserOptions` key. All language options are `false` by default:
```

Comment:
```suggestion
If you are using the built-in ESLint parser, you can additionally change how ESLint interprets your code by specifying the `languageOptions.parserOptions` key. All options are `false` by default:
```

This may sound like it refers to `languageOptions` properties, while it refers to the default parser's options.